src/modules/ce/hooks/useProjectCeVisibility.tsx

```diff
@@ -20,7 +20,8 @@ export function useProjectCeVisibility(project?: ProjectAllFieldsFragment) {
       canViewUnits,
     } = project.access;
 
-    // If the project supports referrals AND the user can view referrals, show the Referrals tab
+    // If the project supports referrals AND the user can view referrals, show the Referrals tab.
+    // This refers to the "Referrals" sub-tab within the referral page, not the project page Referrals tab.
```

Comment:
```suggestion
    // This refers to the "Referrals" sub-tab within the Project Referrals page, not the Project Referrals page itself.
```
are we saying the same thing? I didn't understand referring to the Project Referrals page as a "tab," but maybe I am missing something.

Reply:
Yeah I think we are! I'll try to clarify. I got confused by "showReferrals" thinking it was referring to the left-hand nav. I'll name that explicitly 
<img width="461" height="649" alt="Screenshot 2025-09-03 at 2 26 01 PM" src="https://github.com/user-attachments/assets/249d9457-0de9-49dd-a68a-41fbefae2aaf" />